src/dmd/root/filename.d

```diff
@@ -894,4 +895,82 @@ version(Windows)
 
         return F(ret);
     }
+
+}
+
+version(Posix)
+{
+
+    /**************************
+     * Takes a callable F and applies it to the result of converting
+     * `fileName` to an absolute file path (char*)
+     * Params:
+     *     fileName: The file name to be converted to an absolute path
+     * Returns: Whatever `F` returns.
+     */
```

Comment:
Please remove the "*" prefix, and put a newline above "Params" and "Returns".  See the example at https://dlang.org/dstyle.html#phobos_documentation